tilings/algorithms/gridded_perm_generation.py

```diff
@@ -32,17 +32,20 @@ class GriddedPermsOnTiling:
     built by inserting points into the minimal gridded permutations.
     """
 
-    def __init__(self, tiling: "Tiling"):
+    def __init__(self, tiling: "Tiling", yield_non_minimal=False):
```

Comment:
this should be typed:
`yield_non_minimal: bool = False`
in `new_obs` method too.